pkg/datarepair/queue.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package datarepair
+
+import "storj.io/storj/pkg/pb"
+
+//Queue implements the RepairQueue interface
+type Queue struct {
+	//redis db of repair segments
+	//offline nodes?
+}
+
+//NewQueue creates a new data repair queue
+func NewQueue() {
+
+}
+
+//Add adds a repair segment to the queue
+func (q Queue) Add(qi *pb.QueueItem) error {
```

Comment:
This pb.QueueItem type should match the message type defined in the protobuf, right?

Reply:
oh yes, i missed that, good catch